lib/bandit/http1/adapter.ex

```diff
@@ -28,6 +29,7 @@ defmodule Bandit.HTTP1.Adapter do
           body_remaining: nil | integer(),
           body_encoding: nil | binary(),
           version: nil | :"HTTP/1.1" | :"HTTP/1.0",
+          method: Plug.Conn.method(),
```

Comment:
Since `Plug.Conn.method()` is `binary()`?

```suggestion
          method: nil | Plug.Conn.method(),
```

Reply:
Yup! I did this for http/2 but forgot http/1

Reply:
Done